gateway/src/apicast/policy/ip_check/ip_check.lua

```diff
@@ -0,0 +1,75 @@
+local iputils = require("resty.iputils")
```

Comment:
We should move this into `production` group in the Roverfile:

https://github.com/3scale/apicast/blob/6edf4a088527e2528e30113b75fcf056e4ed1474/gateway/Roverfile#L14

Reply:
I didn't notice this one is in the "development" group :+1:

Reply:
Fixed :+1: